litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/index.tsx

```diff
@@ -197,6 +220,23 @@ const TuneWorkflow: React.FC = () => {
         handleDone={handleDone}
       />
       {/* Experiment Details */}
+      <div className={classes.experimentWrapper}>
+        {/* Edit Button */}
+        <ButtonOutlined>
+          <img src="./icons/editsequence.svg" alt="Edit Sequence" />{' '}
+          <Width width="0.5rem" />
+          Edit Sequence
+        </ButtonOutlined>
+        {/* Details Section -> Graph on the Left and Table on the Right */}
+        <Row>
+          {/* Argo Workflow Graph */}
+          <Width width="30%">Argo Graph</Width>
```

Comment:
And this

Reply:
This is a dummy placeholder that will be replaced by Argo Graph later.